packages/bundle-size/src/utils/readConfig.js

```diff
@@ -0,0 +1,41 @@
+const findUp = require('find-up');
+
+/**
+ * @typedef {{
+ *  webpack(config: import("webpack").Configuration): import("webpack").Configuration
```

Comment:
let's avoid method definitions as they are not properly checked for covariance and such

```suggestion
 *  webpack: (config: import("webpack").Configuration) => import("webpack").Configuration
```